components/datatypes/qoedatadetails.schema.json

```diff
@@ -15,66 +15,86 @@
           "title": "Dropped Frames",
           "type": "integer",
           "description": "The number of frames dropped during playback of the main content."
         },
+        "xdm:isDroppedBeforeStart": {
+          "title": "Drops Before Starts",
+          "type": "boolean",
+          "description": "The number of times a user quit the video before its start. This metric is set to true only if no content was rendered, regardless of ads."
+        },
         "xdm:timeToStart": {
           "title": "Time To Start",
           "type": "integer",
           "description": "Describes the duration (in seconds) passed between video load and start."
         },
-        "xdm:bitrateChange": {
-          "title": "Bitrate Changes",
-          "type": "integer",
-          "description": "The number of bitrate change events that occurred during a playback session."
-        },
-        "xdm:bitrateAverage": {
-          "title": "Average Bitrate",
-          "type": "number",
-          "description": "A weighted average of all bitrate values related to the play duration that occurred during a playback session."
-        },
-        "xdm:bitrateChangeImpactedStreams": {
-          "title": "Bitrate Change Impacted Streams",
-          "type": "number",
-          "description": "The number of streams in which bitrate changes occurred."
-        },
-        "xdm:droppedFrameImpactedStreams": {
-          "title": "Dropped Frame Impacted Streams",
-          "type": "number",
-          "description": "The number of streams in which frames were dropped."
+        "xdm:hasBufferImpactedStreams": {
+          "title": "Buffer Impacted Streams",
+          "type": "boolean",
+          "description": "The number of streams impacted by buffering. This metric is set to true only if at least one buffer event occurred during a playback session."
         },
         "xdm:bufferCount": {
           "title": "Buffer Events",
-          "type": "number",
+          "type": "integer",
           "description": "The number of buffer states that occurred during a playback session."
         },
         "xdm:bufferTime": {
           "title": "Total Buffer Duration",
-          "type": "number",
+          "type": "integer",
           "description": "Describes the total amount of time in seconds, spent buffering."
         },
-        "xdm:buffer": {
-          "title": "Buffer Impacted Streams",
-          "type": "number",
-          "description": "The number of streams impacted by buffering."
-        },
-        "xdm:error": {
+        "xdm:hasErrorImpactedStreams": {
           "title": "Error Impacted Streams",
-          "type": "number",
-          "description": "The number of streams in which an error event occurred."
+          "type": "boolean",
+          "description": "The number of streams in which an error event occurred (i.e., trackError was called during the playback session, and a type=error heartbeat call was generated)."
```

Comment:
If this is boolean , Can you update description reflecting latest changes

Reply:
@prabhum2 Done, thank you!